server/images.go

```diff
@@ -147,22 +148,21 @@ func (m *Model) PostResponseTemplate(p PromptVars) (string, error) {
 }
 
 type ChatHistory struct {
-	Prompts       []PromptVars
-	CurrentImages []api.ImageData
-	LastSystem    string
+	Prompts    []PromptVars
+	LastSystem string
```

Comment:
Unrelated to this PR but I think this can also be determined from `Prompts` vs passed around.